README.MD

```diff
@@ -26,69 +26,66 @@ This lib is supported by every device with a SDK level of at least 14 (Android 4
 
 gradle: add it in your root build.gradle at the end of repositories:
 
-```
+```gradle
 allprojects {
-	repositories {
-		...
-		maven { url 'https://jitpack.io' }
-	}
+    repositories {
+	maven { url 'https://jitpack.io' }
+    }
 }
 ```
 
 
 ##### Step 2. Add this line to your app's dependencies:
 
 
+```gradle
+implementation 'com.github.fede87:StatusBarAlert:2.0.0'
 ```
 
-implementation 'com.fede987:status-bar-alert:1.2.1'
+### Quick Sample
 
+Create a new StatusBarAlert in activities and fragments.
 
-```
+Using Kotlin DSL:
 
-### Quick Sample
+```kotlin
+statusBarAlert {
+    autoHide()
+    dduration(2, TimeUnit.SECONDS)
```

Comment:
shouldnt this be `duration`?

Reply:
Yes sorry misspelled it in the README, however in code it's correctly

Reply:
I've fixed the readme, thank you